tests/gemm/test_bmm_bf16_cutile.py

```
@@ -0,0 +1,97 @@
+
+import pytest
+import torch
+
+from flashinfer.gemm import bmm_bf16
+from flashinfer.utils import get_compute_capability
+
+
+def _cutile_available() -> bool:
+    try:
+        import cuda.tile  # noqa: F401
+    except Exception:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)